connaisseur/admission_request.py

```diff
@@ -0,0 +1,44 @@
+import json
+from jsonschema import validate, ValidationError
+from connaisseur.k8s_object import K8sObject
+from connaisseur.exceptions import InvalidFormatException
+
+
+class AdmissionRequest:
+    SCHEMA_PATH = "/app/connaisseur/res/ad_request_schema.json"
+
+    def __init__(self, ad_request: dict):
+        self.__validate(ad_request)
+
+        request = ad_request["request"]
+        self.uid = request["uid"]
+        self.kind = request["kind"]["kind"]
+        self.namespace = request["namespace"]
+        self.operation = request["operation"]
+        self.user = request["userInfo"]["username"]
+        self.k8s_object = K8sObject(request["object"], self.namespace)
```

Comment:
In a sense, yea, but i don't see much value in turning the `AdmissionRequest` object into a `K8sObject`. It would have a bunch of function it doesn't need...should rename the classes or leave it at that?

Reply:
I changed the `K8sObject`s into `WorkloadObject`s. I think that should resolve it.

Reply:
Yeah, sorry for being unclear, renaming was what I had in mind.